doc/source/howtos/upgrade_guide.rst

```diff
@@ -358,7 +315,13 @@ Scenario: Upgrading a node which is an async replication slave
 
 In this case you need to configure the master-slave replication. Make one of the
 PXC nodes as an async slave. Upgrade the |pxc| async slave node and check the
-replication status.
+replication status. As part of the upgrade, the `RESET SLAVE ALL;` command is run.
```

Comment:
The 'RESET SLAVE ALL' is run as part of an SST.  If you just upgrade a node from 5.7 to 8.0, 'RESET SLAVE ALL' does not get run.